rotkehlchen/data_import/importers/bittrex.py

```diff
@@ -107,19 +178,36 @@ def _import_csv(self, write_cursor: DBCursor, filepath: Path, **kwargs: Any) ->
 
             # by now in both files we should have reached the header
             if 'Date,Currency,Type,Address,Memo/Tag,TxId,Amount' in line:
-                consumer_fn = self._consume_deposits_or_withdrawals
-                save_fn = self.add_asset_movement
+                file_type = BittrexFileType.DEPOSITS_AND_WITHDRAWALS
+            elif 'Id,Amount,Currency,Confirmations,LastUpdated,TxId,CryptoAddress' in line:
+                file_type = BittrexFileType.DEPOSITS
+            elif 'UUID,AMOUNT,CURRENCY,CRYPTOADDRESS,TXID,LASTUPDATED,CONFIRMATIONS,SOURCE,STATE' in line:  # noqa: E501
+                file_type = BittrexFileType.DEPOSITS_OLD
+            elif 'PaymentUuid,Currency,Amount,Address,Opened,Authorized,PendingPayment,TxCost,TxId,Canceled,InvalidAddress' in line:  # noqa: E501
+                file_type = BittrexFileType.WITHDRAWALS
+            elif 'UUID,AMOUNT,CURRENCY,ADDRESS,TXID,OPENED,LASTUPDATED,CLOSED,TXCOST,STATE' in line:  # noqa: E501
+                file_type = BittrexFileType.WITHDRAWALS_OLD
             elif 'TXID,Time (UTC),Transaction,Order Type,Market,Base,Quote,Price,Quantity (Base),Fees (Quote),Total (Quote),Approx Value (USD),Time In Force,Notes' in line:  # noqa: E501
+                file_type = BittrexFileType.TRADES
+            elif 'OrderUuid,Exchange,Type,Quantity,Limit,CommissionPaid,Price,Opened,Closed' in line:  # noqa: E501
+                file_type = BittrexFileType.TRADES_OLD
+            elif 'UUID,QUOTE,MARKET,ORDERTYPE,LIMIT,CEILING,QUANTITY,FILLED,PROCEEDS,COMISSIONPAID,OPENED,CLOSED,TIMEINFORCETYPE,ISCONDITIONAL,CONDITION,CONDITIONTARGET' in line:  # noqa: E501
+                file_type = BittrexFileType.TRADES_OLDER
+            else:
+                raise InputError('The given bittrex CSV file format can not be recognized')
+
+            if file_type in {BittrexFileType.TRADES, BittrexFileType.TRADES_OLD, BittrexFileType.TRADES_OLDER}:  # noqa: E501
```

Comment:
It will take more lines of code, but will reduce logic checks if you add consumer_fn and save_fn in the `if/elif/else` check above right?